modules/payments/modules/payments/README.md

```diff
@@ -1,59 +1,87 @@
+- Then do `pod install`
+- open the `ios/PROJECT_NAME.xcworkspace` file in Xcode.
+- Create a new Swift file to the project `File > New > File > Swift File`, give it any name (e.g. Fix.swift) and create a `bridging header` when prompted by Xcode.
+- select the project and click on `build settings` search for `Library Search Paths` double click values and remove all swift related entries such as: \$(TOOLCHAIN_DIR)/usr/lib/swift/\$(PLATFORM_NAME) and \$(TOOLCHAIN_DIR)/usr/lib/swift-5.0/\$(PLATFORM_NAME)
 
-# Android
-AndroidManifest.xml
-<application>
-...
-<meta-data
-        android:name="com.google.android.gms.wallet.api.enabled"
-        android:value="true" />
-...
-</application>
 
+### **Setup Apple inAPP purchase Local testing environment**
 
-app/build.gradle
+- Create a new file to the project `File > New > File > StoreKit Configuration File`, give it a name (e.g. Configuration).
+- you will see a UI to add products click + icon and choose from options
+  - [Add consumable In-App Purchase](https://developer.apple.com/in-app-purchase/#:~:text=Offer%20extra%20content%20and%20features,directly%20on%20the%20App%20Store.)
+  - [Add Non-Consumable In-App Purchase](https://developer.apple.com/in-app-purchase/#:~:text=Offer%20extra%20content%20and%20features,directly%20on%20the%20App%20Store.)
+  - [Add Non-Renewing Subscription](https://developer.apple.com/in-app-purchase/#:~:text=Offer%20extra%20content%20and%20features,directly%20on%20the%20App%20Store.)
+  - [Add Auto-Renewable Subscription](https://developer.apple.com/in-app-purchase/#:~:text=Offer%20extra%20content%20and%20features,directly%20on%20the%20App%20Store.)
+- goto `Product > Scheme > Edit Scheme` in **StoreKit Configuration dropdown** select the Configuration file you created.
 
-dependencies {
-    ....
+*(not required step)* You can see sample product configurations in the provided file `ios/Configuration.storekit.sample` copy all the data from this file to your created configuration file if you dont want to create your own products.
+   
+### **Setup apple inAPP purchases backend<a name="headin"></a>**
+- visit **http://127.0.0.1:8000/admin/payments/appleiapproduct/**
+- Add new Record with 
+  - **`name`** (any name)
+  - **`Product id`** (e.g. com.temporary.id) this product id for testing would be from storeKit configuration but on PROD/testflight it will come from appstoreconnect dashboard
+
+### Where to give access when user subscribe / upgrade / downgrade
+- look for class `class AppleIAPayment` in `success` block we have to give relevent access to that user. that may be a new user type of adding group or permissions to a user or whatever the project requires.
+
+
+
+# Setup Android
+
+
+in `AndroidManifest.xml` file add following
+
+    <application> 
+        ...
+        <meta-data android:name="com.google.android.gms.wallet.api.enabled"
+                android:value="true" />
+        ...
+    </application>
```

Comment:
Locate the file `AndroidManifest.xml` and add the following to the application section:

```
    <application> 
        ...
        <meta-data android:name="com.google.android.gms.wallet.api.enabled"
                android:value="true" />
        ...
    </application>
```

Reply:
Improve formatting by using code blocks (```).